samples/authentication/bluetooth/central_auth/sample.yaml

```diff
@@ -3,6 +3,7 @@ sample:
   name: Auth Central
 tests:
   sample.central_auth:
+    build_only: true
     harness: bluetooth
-    platform_allow:  qemu_x86
+    platform_allow: nrf52840dk_nrf52840
```

Comment:
For CI, I added the nrf52840dk_nrf52840 board as the only allowed platform.  This is successful if you run twister with --all option like this:  `twister --all samples/authentication/bluetooth/central_auth/sample.central_auth`